src/main/java/com/faforever/client/preferences/LanguageChannel.java

```diff
@@ -0,0 +1,14 @@
+package com.faforever.client.preferences;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
```

Comment:
What about @Value I thinks it makes everything final and adds getter as well as an allargs constucor

Reply:
@Value is not supported for enums, only classes.